content/blog/2020/06/2020-06-03-git-performance-improvement-phase1.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Git Plugin Performance Improvement: Phase-1"
+tags:
+- plugins
+- platform-sig
+- developer
+- performance-improvement
```

Comment:
Just "performance"? Please check role strategy posts from the last year

Reply:
Thanks, I checked, it is just performance.